Library/Formula/walkmod.rb

```diff
@@ -0,0 +1,22 @@
+class Walkmod < Formula
+  desc "Java-based project to apply and share code conventions"
+  homepage "http://www.walkmod.com"
+  url "https://bitbucket.org/rpau/walkmod/downloads/walkmod-2.0-installer.zip"
+  sha256 "06f11d9c4acfe3e3e0d20316a49ad613cbd526368308c68670ce749b472fc35f"
+  depends_on :java
```

Comment:
It looks like from your upstream documentation this requires at least Java 1.7? If so you'll need to change the dep here to:

```ruby
depends_on :java => "1.7+"
```
Should be an empty line beneath the `sha256` as well rather than pushed up against it. Thanks! :)